nixos/modules/services/misc/linux-enable-ir-emitter.nix

```diff
@@ -0,0 +1,58 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
+  cfg = config.services.linux-enable-ir-emitter;
+in
+{
+  options = {
+    services.linux-enable-ir-emitter = {
+      enable = mkEnableOption (mdDoc "Linux Enable IR Emitter");
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.linux-enable-ir-emitter;
+        defaultText = "pkgs.linux-enable-ir-emitter";
+        description = mdDoc ''
+          Package to use for the Linux Enable IR Emitter service.
+        '';
+      };
+
+      device = mkOption {
+        type = types.lines;
+        default = "video2";
+        defaultText = "video2";
+        description = mdDoc ''
+          Emitter device to depend on.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
```

Comment:
Yes. If your webcam is not working, you have to run `sudo linux-enable-ir-emitter configure` and follow the instructions. Then it will save the driver configuration in `/etc/linux-enable-ir-config/video2`. That config is then read by the systemd service and applied at boot.